[0m08:13:00.198204 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x105a43f50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x106886150>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x106887190>]}


============================== 08:13:00.202108 | ad963eed-b469-4f49-b746-545046c8ae37 ==============================
[0m08:13:00.202108 [info ] [MainThread]: Running with dbt=1.9.4
[0m08:13:00.202377 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/joseservin/.dbt', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'debug': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'invocation_command': 'dbt init BigData', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m08:13:00.209470 [debug] [MainThread]: Starter project path: /Users/joseservin/BigDataETLGroup5/venv/lib/python3.11/site-packages/dbt/include/starter_project
[0m08:13:00.212191 [info ] [MainThread]: 
Your new dbt project "BigData" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m08:13:00.212366 [info ] [MainThread]: Setting up your profile.
[0m08:16:35.389103 [info ] [MainThread]: Profile BigData written to /Users/joseservin/.dbt/profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m08:16:35.528274 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 215.35962, "process_in_blocks": "0", "process_kernel_time": 0.129235, "process_mem_max_rss": "122486784", "process_out_blocks": "0", "process_user_time": 0.571338}
[0m08:16:35.529020 [debug] [MainThread]: Command `dbt init` succeeded at 08:16:35.528908 after 215.36 seconds
[0m08:16:35.529492 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10686af50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10681eb10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x106630990>]}
[0m08:16:35.529946 [debug] [MainThread]: Flushing usage events
[0m08:16:35.734250 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m08:16:43.748746 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x109245b50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x109297a50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1092b2650>]}


============================== 08:16:43.750714 | 5f90595b-0287-417c-b8e2-3caaf6fbee0b ==============================
[0m08:16:43.750714 [info ] [MainThread]: Running with dbt=1.9.4
[0m08:16:43.750985 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/joseservin/.dbt', 'debug': 'False', 'version_check': 'True', 'log_path': 'logs', 'fail_fast': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'static_parser': 'True', 'invocation_command': 'dbt debug', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m08:16:43.757392 [info ] [MainThread]: dbt version: 1.9.4
[0m08:16:43.757580 [info ] [MainThread]: python version: 3.11.12
[0m08:16:43.757731 [info ] [MainThread]: python path: /Users/joseservin/BigDataETLGroup5/venv/bin/python
[0m08:16:43.757857 [info ] [MainThread]: os info: macOS-15.4-arm64-arm-64bit
[0m08:16:43.856683 [info ] [MainThread]: Using profiles dir at /Users/joseservin/.dbt
[0m08:16:43.856912 [info ] [MainThread]: Using profiles.yml file at /Users/joseservin/.dbt/profiles.yml
[0m08:16:43.857042 [info ] [MainThread]: Using dbt_project.yml file at /Users/joseservin/BigDataETLGroup5/dbt_project.yml
[0m08:16:43.857194 [error] [MainThread]: Encountered an error:
Internal Error
  Profile should not be None if loading profile completed
[0m08:16:43.858274 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_success": false, "command_wall_clock_time": 0.13416158, "process_in_blocks": "0", "process_kernel_time": 0.099646, "process_mem_max_rss": "118472704", "process_out_blocks": "0", "process_user_time": 0.554319}
[0m08:16:43.858436 [debug] [MainThread]: Command `dbt debug` failed at 08:16:43.858400 after 0.13 seconds
[0m08:16:43.858578 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x109296f50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x109246b90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x108f19ad0>]}
[0m08:16:43.858728 [debug] [MainThread]: Flushing usage events
[0m08:16:44.021780 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m08:18:59.855734 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1053ba310>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10542a050>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10542b090>]}


============================== 08:18:59.857915 | 25a913ac-f93e-4ca5-bccb-7805ef0078d1 ==============================
[0m08:18:59.857915 [info ] [MainThread]: Running with dbt=1.9.4
[0m08:18:59.858191 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/joseservin/.dbt', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'warn_error': 'None', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'introspect': 'True', 'log_format': 'default', 'target_path': 'None', 'invocation_command': 'dbt debug', 'send_anonymous_usage_stats': 'True'}
[0m08:18:59.864800 [info ] [MainThread]: dbt version: 1.9.4
[0m08:18:59.864989 [info ] [MainThread]: python version: 3.11.12
[0m08:18:59.865134 [info ] [MainThread]: python path: /Users/joseservin/BigDataETLGroup5/venv/bin/python
[0m08:18:59.865260 [info ] [MainThread]: os info: macOS-15.4-arm64-arm-64bit
[0m08:18:59.975677 [info ] [MainThread]: Using profiles dir at /Users/joseservin/.dbt
[0m08:18:59.975922 [info ] [MainThread]: Using profiles.yml file at /Users/joseservin/.dbt/profiles.yml
[0m08:18:59.976058 [info ] [MainThread]: Using dbt_project.yml file at /Users/joseservin/BigDataETLGroup5/dbt_project.yml
[0m08:18:59.976213 [error] [MainThread]: Encountered an error:
Internal Error
  Profile should not be None if loading profile completed
[0m08:18:59.977755 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_success": false, "command_wall_clock_time": 0.14693442, "process_in_blocks": "0", "process_kernel_time": 0.110273, "process_mem_max_rss": "119832576", "process_out_blocks": "0", "process_user_time": 0.550064}
[0m08:18:59.977933 [debug] [MainThread]: Command `dbt debug` failed at 08:18:59.977899 after 0.15 seconds
[0m08:18:59.978077 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1053ba890>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10542a510>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x100f01190>]}
[0m08:18:59.978228 [debug] [MainThread]: Flushing usage events
[0m08:19:00.166196 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m08:22:07.476908 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10771d6d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x107774bd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10776ef10>]}


============================== 08:22:07.479140 | 27f6740f-0b1b-40a4-a357-5ba0c29e8411 ==============================
[0m08:22:07.479140 [info ] [MainThread]: Running with dbt=1.9.4
[0m08:22:07.479419 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'version_check': 'True', 'log_path': 'logs', 'fail_fast': 'False', 'profiles_dir': '/Users/joseservin/BigDataETLGroup5/BigData/profiles.yml', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt debug', 'introspect': 'True', 'static_parser': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m08:22:07.486169 [info ] [MainThread]: dbt version: 1.9.4
[0m08:22:07.486324 [info ] [MainThread]: python version: 3.11.12
[0m08:22:07.486462 [info ] [MainThread]: python path: /Users/joseservin/BigDataETLGroup5/venv/bin/python
[0m08:22:07.486586 [info ] [MainThread]: os info: macOS-15.4-arm64-arm-64bit
[0m08:22:07.486721 [info ] [MainThread]: Using profiles dir at /Users/joseservin/BigDataETLGroup5/BigData/profiles.yml
[0m08:22:07.486852 [info ] [MainThread]: Using profiles.yml file at /Users/joseservin/BigDataETLGroup5/BigData/profiles.yml/profiles.yml
[0m08:22:07.486967 [info ] [MainThread]: Using dbt_project.yml file at /Users/joseservin/BigDataETLGroup5/dbt_project.yml
[0m08:22:07.487083 [info ] [MainThread]: Configuration:
[0m08:22:07.487190 [info ] [MainThread]:   profiles.yml file [[31mERROR not found[0m]
[0m08:22:07.487298 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m08:22:07.487402 [info ] [MainThread]: Required dependencies:
[0m08:22:07.487531 [debug] [MainThread]: Executing "git --help"
[0m08:22:07.496782 [debug] [MainThread]: STDOUT: "b"usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]\n           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]\n           [--super-prefix=<path>] [--config-env=<name>=<envvar>]\n           <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone     Clone a repository into a new directory\n   init      Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add       Add file contents to the index\n   mv        Move or rename a file, a directory, or a symlink\n   restore   Restore working tree files\n   rm        Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect    Use binary search to find the commit that introduced a bug\n   diff      Show changes between commits, commit and working tree, etc\n   grep      Print lines matching a pattern\n   log       Show commit logs\n   show      Show various types of objects\n   status    Show the working tree status\n\ngrow, mark and tweak your common history\n   branch    List, create, or delete branches\n   commit    Record changes to the repository\n   merge     Join two or more development histories together\n   rebase    Reapply commits on top of another base tip\n   reset     Reset current HEAD to the specified state\n   switch    Switch branches\n   tag       Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch     Download objects and refs from another repository\n   pull      Fetch from and integrate with another repository or a local branch\n   push      Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m08:22:07.497226 [debug] [MainThread]: STDERR: "b''"
[0m08:22:07.497387 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m08:22:07.497535 [info ] [MainThread]: Connection test skipped since no profile was found
[0m08:22:07.497721 [info ] [MainThread]: [31m2 checks failed:[0m
[0m08:22:07.497848 [info ] [MainThread]: dbt looked for a profiles.yml file in /Users/joseservin/BigDataETLGroup5/BigData/profiles.yml/profiles.yml, but did
not find one. For more information on configuring your profile, consult the
documentation:

https://docs.getdbt.com/docs/configure-your-profile


[0m08:22:07.497973 [info ] [MainThread]: Project loading failed for the following reason:
 project path </Users/joseservin/BigDataETLGroup5/dbt_project.yml> not found

[0m08:22:07.499621 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_success": false, "command_wall_clock_time": 0.04730421, "process_in_blocks": "0", "process_kernel_time": 0.094957, "process_mem_max_rss": "102629376", "process_out_blocks": "0", "process_user_time": 0.486608}
[0m08:22:07.499861 [debug] [MainThread]: Command `dbt debug` failed at 08:22:07.499814 after 0.05 seconds
[0m08:22:07.500077 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10771ead0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10779a3d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x107799610>]}
[0m08:22:07.500269 [debug] [MainThread]: Flushing usage events
[0m08:22:07.717799 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m08:22:36.665479 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10ad18bd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10a8f1b50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10ad86bd0>]}


============================== 08:22:36.667037 | 43f95690-a9eb-4f8e-82a8-19870ea25194 ==============================
[0m08:22:36.667037 [info ] [MainThread]: Running with dbt=1.9.4
[0m08:22:36.667310 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/joseservin/BigDataETLGroup5/BigData/', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'debug': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'introspect': 'True', 'invocation_command': 'dbt debug', 'static_parser': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m08:22:36.671485 [info ] [MainThread]: dbt version: 1.9.4
[0m08:22:36.671648 [info ] [MainThread]: python version: 3.11.12
[0m08:22:36.671789 [info ] [MainThread]: python path: /Users/joseservin/BigDataETLGroup5/venv/bin/python
[0m08:22:36.671917 [info ] [MainThread]: os info: macOS-15.4-arm64-arm-64bit
[0m08:22:36.672419 [info ] [MainThread]: target not specified in profile 'BigData', using 'default'
[0m08:22:36.672611 [info ] [MainThread]: Using profiles dir at /Users/joseservin/BigDataETLGroup5/BigData/
[0m08:22:36.672730 [info ] [MainThread]: Using profiles.yml file at /Users/joseservin/BigDataETLGroup5/BigData/profiles.yml
[0m08:22:36.672850 [info ] [MainThread]: Using dbt_project.yml file at /Users/joseservin/BigDataETLGroup5/dbt_project.yml
[0m08:22:36.672963 [info ] [MainThread]: Configuration:
[0m08:22:36.673077 [info ] [MainThread]:   profiles.yml file [[31mERROR invalid[0m]
[0m08:22:36.673181 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m08:22:36.673299 [info ] [MainThread]: Required dependencies:
[0m08:22:36.673424 [debug] [MainThread]: Executing "git --help"
[0m08:22:36.682114 [debug] [MainThread]: STDOUT: "b"usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]\n           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]\n           [--super-prefix=<path>] [--config-env=<name>=<envvar>]\n           <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone     Clone a repository into a new directory\n   init      Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add       Add file contents to the index\n   mv        Move or rename a file, a directory, or a symlink\n   restore   Restore working tree files\n   rm        Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect    Use binary search to find the commit that introduced a bug\n   diff      Show changes between commits, commit and working tree, etc\n   grep      Print lines matching a pattern\n   log       Show commit logs\n   show      Show various types of objects\n   status    Show the working tree status\n\ngrow, mark and tweak your common history\n   branch    List, create, or delete branches\n   commit    Record changes to the repository\n   merge     Join two or more development histories together\n   rebase    Reapply commits on top of another base tip\n   reset     Reset current HEAD to the specified state\n   switch    Switch branches\n   tag       Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch     Download objects and refs from another repository\n   pull      Fetch from and integrate with another repository or a local branch\n   push      Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m08:22:36.682476 [debug] [MainThread]: STDERR: "b''"
[0m08:22:36.682603 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m08:22:36.682740 [info ] [MainThread]: Connection test skipped since no profile was found
[0m08:22:36.682912 [info ] [MainThread]: [31m2 checks failed:[0m
[0m08:22:36.683033 [info ] [MainThread]: Could not load dbt_project.yml
Profile loading failed for the following reason:
Runtime Error
  The profile 'BigData' does not have a target named 'default'. The valid target names for this profile are:
   - dev


[0m08:22:36.683160 [info ] [MainThread]: Project loading failed for the following reason:
 project path </Users/joseservin/BigDataETLGroup5/dbt_project.yml> not found

[0m08:22:36.684286 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_success": false, "command_wall_clock_time": 0.042845625, "process_in_blocks": "0", "process_kernel_time": 0.082717, "process_mem_max_rss": "102776832", "process_out_blocks": "0", "process_user_time": 0.481867}
[0m08:22:36.684464 [debug] [MainThread]: Command `dbt debug` failed at 08:22:36.684425 after 0.04 seconds
[0m08:22:36.684661 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10ad86ad0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10ad1ac10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10a8f1b90>]}
[0m08:22:36.684837 [debug] [MainThread]: Flushing usage events
[0m08:22:36.853868 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m08:24:20.547425 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x111d191d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x111d85cd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x111d86d10>]}


============================== 08:24:20.549641 | 2aadeebd-7bc4-48cf-8604-4525cbab1ced ==============================
[0m08:24:20.549641 [info ] [MainThread]: Running with dbt=1.9.4
[0m08:24:20.549919 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'debug': 'False', 'profiles_dir': '/Users/joseservin/BigDataETLGroup5/BigData/', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt debug', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m08:24:20.556954 [info ] [MainThread]: dbt version: 1.9.4
[0m08:24:20.557115 [info ] [MainThread]: python version: 3.11.12
[0m08:24:20.557259 [info ] [MainThread]: python path: /Users/joseservin/BigDataETLGroup5/venv/bin/python
[0m08:24:20.557382 [info ] [MainThread]: os info: macOS-15.4-arm64-arm-64bit
[0m08:24:20.676183 [info ] [MainThread]: Using profiles dir at /Users/joseservin/BigDataETLGroup5/BigData/
[0m08:24:20.676419 [info ] [MainThread]: Using profiles.yml file at /Users/joseservin/BigDataETLGroup5/BigData/profiles.yml
[0m08:24:20.676555 [info ] [MainThread]: Using dbt_project.yml file at /Users/joseservin/BigDataETLGroup5/dbt_project.yml
[0m08:24:20.676886 [info ] [MainThread]: adapter type: snowflake
[0m08:24:20.677011 [info ] [MainThread]: adapter version: 1.9.0
[0m08:24:20.677132 [info ] [MainThread]: Configuration:
[0m08:24:20.677246 [info ] [MainThread]:   profiles.yml file [[32mOK found and valid[0m]
[0m08:24:20.677354 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m08:24:20.677462 [info ] [MainThread]: Required dependencies:
[0m08:24:20.677592 [debug] [MainThread]: Executing "git --help"
[0m08:24:20.687922 [debug] [MainThread]: STDOUT: "b"usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]\n           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]\n           [--super-prefix=<path>] [--config-env=<name>=<envvar>]\n           <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone     Clone a repository into a new directory\n   init      Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add       Add file contents to the index\n   mv        Move or rename a file, a directory, or a symlink\n   restore   Restore working tree files\n   rm        Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect    Use binary search to find the commit that introduced a bug\n   diff      Show changes between commits, commit and working tree, etc\n   grep      Print lines matching a pattern\n   log       Show commit logs\n   show      Show various types of objects\n   status    Show the working tree status\n\ngrow, mark and tweak your common history\n   branch    List, create, or delete branches\n   commit    Record changes to the repository\n   merge     Join two or more development histories together\n   rebase    Reapply commits on top of another base tip\n   reset     Reset current HEAD to the specified state\n   switch    Switch branches\n   tag       Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch     Download objects and refs from another repository\n   pull      Fetch from and integrate with another repository or a local branch\n   push      Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m08:24:20.688544 [debug] [MainThread]: STDERR: "b''"
[0m08:24:20.688696 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m08:24:20.688867 [info ] [MainThread]: Connection:
[0m08:24:20.689033 [info ] [MainThread]:   account: EOTZXXO-GS61999
[0m08:24:20.689158 [info ] [MainThread]:   user: ALEXMBE96
[0m08:24:20.689273 [info ] [MainThread]:   database: DW
[0m08:24:20.689383 [info ] [MainThread]:   warehouse: COMPUTE_WH
[0m08:24:20.689489 [info ] [MainThread]:   role: ACCOUNTADMIN
[0m08:24:20.689595 [info ] [MainThread]:   schema: DVD_RENTAL
[0m08:24:20.689701 [info ] [MainThread]:   authenticator: None
[0m08:24:20.689820 [info ] [MainThread]:   oauth_client_id: None
[0m08:24:20.689930 [info ] [MainThread]:   query_tag: None
[0m08:24:20.690037 [info ] [MainThread]:   client_session_keep_alive: False
[0m08:24:20.690141 [info ] [MainThread]:   host: None
[0m08:24:20.690248 [info ] [MainThread]:   port: None
[0m08:24:20.690355 [info ] [MainThread]:   proxy_host: None
[0m08:24:20.690461 [info ] [MainThread]:   proxy_port: None
[0m08:24:20.690565 [info ] [MainThread]:   protocol: None
[0m08:24:20.690674 [info ] [MainThread]:   connect_retries: 1
[0m08:24:20.690780 [info ] [MainThread]:   connect_timeout: None
[0m08:24:20.690887 [info ] [MainThread]:   retry_on_database_errors: False
[0m08:24:20.690992 [info ] [MainThread]:   retry_all: False
[0m08:24:20.691094 [info ] [MainThread]:   insecure_mode: False
[0m08:24:20.691197 [info ] [MainThread]:   reuse_connections: True
[0m08:24:20.691472 [info ] [MainThread]: Registered adapter: snowflake=1.9.0
[0m08:24:20.893896 [debug] [MainThread]: Acquiring new snowflake connection 'debug'
[0m08:24:20.913577 [debug] [MainThread]: Using snowflake connection "debug"
[0m08:24:20.913750 [debug] [MainThread]: On debug: select 1 as id
[0m08:24:20.913869 [debug] [MainThread]: Opening a new connection, currently in state init
[0m08:24:21.794626 [debug] [MainThread]: SQL status: SUCCESS 1 in 0.880 seconds
[0m08:24:21.797619 [info ] [MainThread]:   Connection test: [[32mOK connection ok[0m]

[0m08:24:21.798363 [info ] [MainThread]: [31m1 check failed:[0m
[0m08:24:21.798969 [info ] [MainThread]: Project loading failed for the following reason:
 project path </Users/joseservin/BigDataETLGroup5/dbt_project.yml> not found

[0m08:24:21.805500 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_success": false, "command_wall_clock_time": 1.2841899, "process_in_blocks": "0", "process_kernel_time": 0.179247, "process_mem_max_rss": "139231232", "process_out_blocks": "0", "process_user_time": 0.769114}
[0m08:24:21.806235 [debug] [MainThread]: Command `dbt debug` failed at 08:24:21.806094 after 1.29 seconds
[0m08:24:21.806865 [debug] [MainThread]: Connection 'debug' was left open.
[0m08:24:21.807283 [debug] [MainThread]: On debug: Close
[0m08:24:21.909742 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x113396dd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x111d1abd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x114463f90>]}
[0m08:24:21.910390 [debug] [MainThread]: Flushing usage events
[0m08:24:22.159325 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m19:44:12.293060 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x112c19590>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x112c27b50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x112c27f90>]}


============================== 19:44:12.295477 | 88c20753-e495-4a02-8446-e40346229716 ==============================
[0m19:44:12.295477 [info ] [MainThread]: Running with dbt=1.9.4
[0m19:44:12.295772 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/joseservin/BigDataETLGroup5/BigData/', 'debug': 'False', 'version_check': 'True', 'log_path': 'logs', 'fail_fast': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'static_parser': 'True', 'invocation_command': 'dbt debug', 'introspect': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m19:44:12.302821 [info ] [MainThread]: dbt version: 1.9.4
[0m19:44:12.302965 [info ] [MainThread]: python version: 3.11.12
[0m19:44:12.303090 [info ] [MainThread]: python path: /Users/joseservin/BigDataETLGroup5/venv/bin/python
[0m19:44:12.303204 [info ] [MainThread]: os info: macOS-15.4-arm64-arm-64bit
[0m19:44:12.303474 [error] [MainThread]: Encountered an error:
Runtime Error
  Syntax error near line 10
  ------------------------------
  7  |       role: ACCOUNTADMIN
  8  |       schema: DVD_RENTAL
  9  |       threads: 1
  10 |       type: snowflake user: ALEXMBE96
  11 |       warehouse: COMPUTE_WH
  12 | target: dev
  
  Raw Error:
  ------------------------------
  mapping values are not allowed in this context
    in "<unicode string>", line 10, column 27
[0m19:44:12.307867 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_success": false, "command_wall_clock_time": 0.027111333, "process_in_blocks": "0", "process_kernel_time": 0.124732, "process_mem_max_rss": "102400000", "process_out_blocks": "0", "process_user_time": 0.515812}
[0m19:44:12.308081 [debug] [MainThread]: Command `dbt debug` failed at 19:44:12.308042 after 0.03 seconds
[0m19:44:12.308236 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x112c1aa50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x112c27a50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104c87f90>]}
[0m19:44:12.308405 [debug] [MainThread]: Flushing usage events
[0m19:44:12.540949 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m19:44:27.366194 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10484b4d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104861f50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10484acd0>]}


============================== 19:44:27.367871 | abc296dd-eadc-4624-b7a2-c5b241ba76fe ==============================
[0m19:44:27.367871 [info ] [MainThread]: Running with dbt=1.9.4
[0m19:44:27.368111 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/joseservin/BigDataETLGroup5/BigData/', 'fail_fast': 'False', 'warn_error': 'None', 'log_path': 'logs', 'version_check': 'True', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'static_parser': 'True', 'invocation_command': 'dbt debug', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m19:44:27.372027 [info ] [MainThread]: dbt version: 1.9.4
[0m19:44:27.372175 [info ] [MainThread]: python version: 3.11.12
[0m19:44:27.372303 [info ] [MainThread]: python path: /Users/joseservin/BigDataETLGroup5/venv/bin/python
[0m19:44:27.372425 [info ] [MainThread]: os info: macOS-15.4-arm64-arm-64bit
[0m19:44:27.372877 [info ] [MainThread]: target not specified in profile 'BigData', using 'default'
[0m19:44:27.373113 [info ] [MainThread]: target not specified in profile 'target', using 'default'
[0m19:44:27.373256 [info ] [MainThread]: Using profiles dir at /Users/joseservin/BigDataETLGroup5/BigData/
[0m19:44:27.373364 [info ] [MainThread]: Using profiles.yml file at /Users/joseservin/BigDataETLGroup5/BigData/profiles.yml
[0m19:44:27.373467 [info ] [MainThread]: Using dbt_project.yml file at /Users/joseservin/BigDataETLGroup5/dbt_project.yml
[0m19:44:27.373569 [info ] [MainThread]: Configuration:
[0m19:44:27.373668 [info ] [MainThread]:   profiles.yml file [[31mERROR invalid[0m]
[0m19:44:27.373764 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m19:44:27.373860 [info ] [MainThread]: Required dependencies:
[0m19:44:27.373972 [debug] [MainThread]: Executing "git --help"
[0m19:44:27.383072 [debug] [MainThread]: STDOUT: "b"usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]\n           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]\n           [--super-prefix=<path>] [--config-env=<name>=<envvar>]\n           <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone     Clone a repository into a new directory\n   init      Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add       Add file contents to the index\n   mv        Move or rename a file, a directory, or a symlink\n   restore   Restore working tree files\n   rm        Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect    Use binary search to find the commit that introduced a bug\n   diff      Show changes between commits, commit and working tree, etc\n   grep      Print lines matching a pattern\n   log       Show commit logs\n   show      Show various types of objects\n   status    Show the working tree status\n\ngrow, mark and tweak your common history\n   branch    List, create, or delete branches\n   commit    Record changes to the repository\n   merge     Join two or more development histories together\n   rebase    Reapply commits on top of another base tip\n   reset     Reset current HEAD to the specified state\n   switch    Switch branches\n   tag       Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch     Download objects and refs from another repository\n   pull      Fetch from and integrate with another repository or a local branch\n   push      Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m19:44:27.383416 [debug] [MainThread]: STDERR: "b''"
[0m19:44:27.383535 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m19:44:27.383658 [info ] [MainThread]: Connection test skipped since no profile was found
[0m19:44:27.383806 [info ] [MainThread]: [31m2 checks failed:[0m
[0m19:44:27.383917 [info ] [MainThread]: Could not load dbt_project.yml
Profile loading failed for the following reason:
Runtime Error
  The profile 'BigData' does not have a target named 'default'. The valid target names for this profile are:
   - dev

Runtime Error
  outputs not specified in profile 'target'


[0m19:44:27.384026 [info ] [MainThread]: Project loading failed for the following reason:
 project path </Users/joseservin/BigDataETLGroup5/dbt_project.yml> not found

[0m19:44:27.385122 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_success": false, "command_wall_clock_time": 0.042641375, "process_in_blocks": "0", "process_kernel_time": 0.089235, "process_mem_max_rss": "102187008", "process_out_blocks": "0", "process_user_time": 0.487728}
[0m19:44:27.385297 [debug] [MainThread]: Command `dbt debug` failed at 19:44:27.385259 after 0.04 seconds
[0m19:44:27.385495 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104883890>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104883f50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1047f9550>]}
[0m19:44:27.385676 [debug] [MainThread]: Flushing usage events
[0m19:44:27.555881 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m19:45:03.523398 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104dc0050>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104df5d90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104df6e10>]}


============================== 19:45:03.525572 | 32654e4a-0db8-4b62-9e18-64c2db9a4b01 ==============================
[0m19:45:03.525572 [info ] [MainThread]: Running with dbt=1.9.4
[0m19:45:03.525824 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'debug': 'False', 'profiles_dir': '/Users/joseservin/BigDataETLGroup5/BigData/', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'introspect': 'True', 'static_parser': 'True', 'invocation_command': 'dbt debug', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m19:45:03.532696 [info ] [MainThread]: dbt version: 1.9.4
[0m19:45:03.532868 [info ] [MainThread]: python version: 3.11.12
[0m19:45:03.533019 [info ] [MainThread]: python path: /Users/joseservin/BigDataETLGroup5/venv/bin/python
[0m19:45:03.533135 [info ] [MainThread]: os info: macOS-15.4-arm64-arm-64bit
[0m19:45:03.962111 [info ] [MainThread]: Using profiles dir at /Users/joseservin/BigDataETLGroup5/BigData/
[0m19:45:03.962328 [info ] [MainThread]: Using profiles.yml file at /Users/joseservin/BigDataETLGroup5/BigData/profiles.yml
[0m19:45:03.962450 [info ] [MainThread]: Using dbt_project.yml file at /Users/joseservin/BigDataETLGroup5/dbt_project.yml
[0m19:45:03.962655 [info ] [MainThread]: adapter type: snowflake
[0m19:45:03.962774 [info ] [MainThread]: adapter version: 1.9.0
[0m19:45:03.962887 [info ] [MainThread]: Configuration:
[0m19:45:03.962992 [info ] [MainThread]:   profiles.yml file [[32mOK found and valid[0m]
[0m19:45:03.963092 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m19:45:03.963187 [info ] [MainThread]: Required dependencies:
[0m19:45:03.963307 [debug] [MainThread]: Executing "git --help"
[0m19:45:03.972571 [debug] [MainThread]: STDOUT: "b"usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]\n           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]\n           [--super-prefix=<path>] [--config-env=<name>=<envvar>]\n           <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone     Clone a repository into a new directory\n   init      Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add       Add file contents to the index\n   mv        Move or rename a file, a directory, or a symlink\n   restore   Restore working tree files\n   rm        Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect    Use binary search to find the commit that introduced a bug\n   diff      Show changes between commits, commit and working tree, etc\n   grep      Print lines matching a pattern\n   log       Show commit logs\n   show      Show various types of objects\n   status    Show the working tree status\n\ngrow, mark and tweak your common history\n   branch    List, create, or delete branches\n   commit    Record changes to the repository\n   merge     Join two or more development histories together\n   rebase    Reapply commits on top of another base tip\n   reset     Reset current HEAD to the specified state\n   switch    Switch branches\n   tag       Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch     Download objects and refs from another repository\n   pull      Fetch from and integrate with another repository or a local branch\n   push      Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m19:45:03.972968 [debug] [MainThread]: STDERR: "b''"
[0m19:45:03.973115 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m19:45:03.973253 [info ] [MainThread]: Connection:
[0m19:45:03.973389 [info ] [MainThread]:   account: EOTZXXO-GS61999
[0m19:45:03.973491 [info ] [MainThread]:   user: ALEXMBE96
[0m19:45:03.973591 [info ] [MainThread]:   database: DW
[0m19:45:03.973688 [info ] [MainThread]:   warehouse: COMPUTE_WH
[0m19:45:03.973786 [info ] [MainThread]:   role: ACCOUNTADMIN
[0m19:45:03.973879 [info ] [MainThread]:   schema: DVD_RENTAL
[0m19:45:03.973973 [info ] [MainThread]:   authenticator: None
[0m19:45:03.974072 [info ] [MainThread]:   oauth_client_id: None
[0m19:45:03.974164 [info ] [MainThread]:   query_tag: None
[0m19:45:03.974261 [info ] [MainThread]:   client_session_keep_alive: False
[0m19:45:03.974362 [info ] [MainThread]:   host: None
[0m19:45:03.974457 [info ] [MainThread]:   port: None
[0m19:45:03.974549 [info ] [MainThread]:   proxy_host: None
[0m19:45:03.974645 [info ] [MainThread]:   proxy_port: None
[0m19:45:03.974738 [info ] [MainThread]:   protocol: None
[0m19:45:03.974830 [info ] [MainThread]:   connect_retries: 1
[0m19:45:03.974923 [info ] [MainThread]:   connect_timeout: None
[0m19:45:03.975013 [info ] [MainThread]:   retry_on_database_errors: False
[0m19:45:03.975106 [info ] [MainThread]:   retry_all: False
[0m19:45:03.975195 [info ] [MainThread]:   insecure_mode: False
[0m19:45:03.975287 [info ] [MainThread]:   reuse_connections: True
[0m19:45:03.975518 [info ] [MainThread]: Registered adapter: snowflake=1.9.0
[0m19:45:04.019824 [debug] [MainThread]: Acquiring new snowflake connection 'debug'
[0m19:45:04.045267 [debug] [MainThread]: Using snowflake connection "debug"
[0m19:45:04.045444 [debug] [MainThread]: On debug: select 1 as id
[0m19:45:04.045561 [debug] [MainThread]: Opening a new connection, currently in state init
[0m19:45:05.034490 [debug] [MainThread]: SQL status: SUCCESS 1 in 0.989 seconds
[0m19:45:05.038229 [info ] [MainThread]:   Connection test: [[32mOK connection ok[0m]

[0m19:45:05.039344 [info ] [MainThread]: [31m1 check failed:[0m
[0m19:45:05.039936 [info ] [MainThread]: Project loading failed for the following reason:
 project path </Users/joseservin/BigDataETLGroup5/dbt_project.yml> not found

[0m19:45:05.046223 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_success": false, "command_wall_clock_time": 1.5472506, "process_in_blocks": "0", "process_kernel_time": 0.1741, "process_mem_max_rss": "140771328", "process_out_blocks": "0", "process_user_time": 0.757427}
[0m19:45:05.046982 [debug] [MainThread]: Command `dbt debug` failed at 19:45:05.046805 after 1.55 seconds
[0m19:45:05.047579 [debug] [MainThread]: Connection 'debug' was left open.
[0m19:45:05.047997 [debug] [MainThread]: On debug: Close
[0m19:45:05.234133 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104d92950>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104df4e50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104df6690>]}
[0m19:45:05.235998 [debug] [MainThread]: Flushing usage events
[0m19:45:05.414081 [debug] [MainThread]: An error was encountered while trying to flush usage events
